testSrc/com/jetbrains/python/PythonMockSdk.java

```diff
@@ -1,4 +1,4 @@
-// Copyright 2000-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+// Copyright 2000-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
```

Comment:
Why is this file Copyright JetBrains? If this is JetBrains' file we need to bundle the mentioned `LICENSE` file

Reply:
I'm sorry. I forget to add the file.